doc/playbook.rst

```diff
@@ -127,6 +127,15 @@ It accepts messages and gateways them to Cloud Run.
 tester
 ======
 
+Run the tester either on a local machine, or in Cloud Shell.  
+For the latter, install the protype code and the GCP pubsub client:
+
+.. code-block:: sh
+
+    gcloud config set project prompt-proto
+    git clone https://github.com/lsst-dm/prompt_prototype.git
+    pip3 install google-cloud-pubsub
+
 ``tester/upload.py`` is a script that simulates the CCS image writer.
 It takes a JSON token in ``./prompt-proto-upload.json``.
```

Comment:
We should modify it to not force the use of a JSON token when it's not necessary (such as in the Cloud Shell environment).